src/kili/services/asset_import/base.py

```diff
@@ -459,9 +468,10 @@ def import_assets_by_batch(
         self.pbar.total = len(assets)
         self.pbar.refresh()
 
-        responses = []
+        created_asset_ids = []
         for i, batch_assets in enumerate(batch_generator):
             # check last batch only
             verify = i == (len(batch_generator) - 1) and self.verify
-            responses.append(batch_importer.import_batch(batch_assets, verify))
-        return responses[-1]
+            batch_created_asset_ids = batch_importer.import_batch(batch_assets, verify)
+            created_asset_ids.append(batch_created_asset_ids)
```

Comment:
extend instead  of append? We want a flat list imho

Reply:
maybe worth doing a test by mocking the resolver answer

Reply:
yes you are right!